### ISSUE TITLE
Last release is not signed

### ISSUE THREAD

**Issue by IzzySoft** [AmanoTeam/WebInspector#1]
Looks like you forgot to sign today's release (v0.4). Could you please do so? Otherwise the APK cannot be installed by the "average Jane and Joe". Thanks!

**Comment by SnwMds** [comment]
Oh! Thanks for pointing this! I replaced the unsigned APK with a signed one.

**Comment by IzzySoft** [comment]
Thanks a lot! Confirmed solution :smiley: Btw: your app is [available here](https://apt.izzysoft.de/fdroid/index/apk/com.amanoteam.webinspector). Updates are fetched automatically within 24h of your making them available – which is how I encountered the issue (updater complained). The repo can be easily used with any [F-Droid client](https://android.izzysoft.de/applists/category/named/apps_markets#group_1181), which then also informs the user whenever an update becomes available. If you want to link there (e.g. from your Readme here), be welcome to use [my badges](https://gitlab.com/IzzyOnDroid/repo/#if-my-app-is-available-here-do-you-have-a-badge-i-can-use-to-link-to-it).

Thanks for making your app available as FOSS!
